colossalai/inference/core/async_engine.py

```diff
@@ -63,7 +63,7 @@ class RequestTracker:
     """Synchronous abstraction for tracking requests."""
 
     def __init__(self) -> None:
-        self._request_streams: Dict[str, AsyncStream] = {}
+        self._request_streams: Dict[int, AsyncStream] = {}
         self._finished_requests: asyncio.Queue[int] = asyncio.Queue()
```

Comment:
Would this mean that each request will have its own stream which creates a new `asyncio.Queue()`？

Reply:
Actually right

Reply:
I think it might be quite inefficient? Do you think there is better implementation?

Reply:
it is true, but change this would cause a different response, I will check that